pkg/securitycontextconstraints/sccadmission/admission.go

```diff
@@ -440,9 +441,17 @@ func logProviders(pod *coreapi.Pod, providers []sccmatching.SecurityContextConst
 	for i, p := range providers {
 		names[i] = p.GetSCCName()
 	}
-	klog.V(4).Infof("validating pod %s (generate: %s) against providers %s", pod.Name, pod.GenerateName, strings.Join(names, ","))
+	klog.V(2).Infof("validating pod %s (generate: %s) against providers %s", pod.Name, pod.GenerateName, strings.Join(names, ","))
```

Comment:
We can't make this a V(2), this would log every time the admission is triggered. Or is this just for debugging purposes?

Reply:
this is just for debugging purposes, temporarily. But given the finding we can leave this out for good.